site/content/events/2015-denver/_team_global.txt

```diff
@@ -1,13 +1,5 @@
-- Patrick Debois
-- John Willis
-- Damon Edwards
-- Kris Buytaert
-- John Vincent
-- Andrew Shafer
-- Lindsay Holmwood
-- Stephen Nelson-Smith
-- Christian Trabold
-- James Wickett
-- Anthony Goddard
-- Bernd Erk
-- Bridget Kromhout
+- Tavis Aitken
```

Comment:
> Ahh I thought that was a copy paste error. Why is the global team in each event? 
> I can fix it just thought it was odd.

Agreed that it's odd. There are website overhaul things I'd like to do soon that will involve taking the global team out of each event's template; just haven't had time yet. Thanks.